site2/docs/admin-api-overview.md

```diff
@@ -70,3 +70,23 @@ config.setTlsTrustCertsFilePath(tlsTrustCertsFilePath);
 
 PulsarAdmin admin = new PulsarAdmin(url, config);
 ```
+
+If you have mutilple brokers to using, you can use multi host like pulsar service, like this:
+```java
+URL url = new URL("http://localhost:8080,localhost:8081,localhost:8082");
+// Pass auth-plugin class fully-qualified name if Pulsar-security enabled
```

Comment:
```suggestion
// Pass auth-plugin class fully-qualified name if Pulsar-security is enabled.
```

Reply:
Are "auth-plugin" and "Pulsar-security" something like a prop noun?

Reply:
I think not.